python/cucim/src/cucim/skimage/restoration/deconvolution.py

```diff
@@ -307,7 +307,7 @@ def unsupervised_wiener(
     prev_x_postmean = cp.zeros(trans_fct.shape, dtype=float_type)
 
     # Difference between two successive mean
-    delta = np.NAN
+    delta = np.nan
```

Comment:
Wonder if we should use `float("nan")`

Reply:
I don't think there is a strong reason to prefer one or the other in this case. It is only used in a stopping criterion check below

```py
delta < params["threshold"]
```
which should always be False for either